docs/core/deploying/native-aot/index.md

```diff
@@ -103,6 +103,7 @@ Native AOT applications come with a few fundamental limitations and compatibilit
 - Requires trimming, which has [limitations](../trimming/incompatibilities.md).
 - Implies compilation into a single file, which has known [incompatibilities](../single-file/overview.md#api-incompatibility).
 - Apps include required runtime libraries (just like [self-contained apps](../index.md#publish-self-contained), increasing their size as compared to framework-dependent apps).
+- [System.Linq.Expressions](https://learn.microsoft.com/en-us/dotnet/api/system.linq.expressions?view=net-7.0) are interpreted, not compiled, and therefore are much slower than in CoreCLR
```

Comment:
```suggestion
- [System.Linq.Expressions](https://learn.microsoft.com/dotnet/api/system.linq.expressions) always use interpreted form, that is much slower than run-time generated compiled code.
```
- Omit `en-us` and version number in the link
- S.L.Expressions terminology is that you can `Compile` the expression tree to either to interpreted form or compiled code.
- We try to avoid presenting native AOT as something different than CoreCLR.